pkgs/applications/office/dspdfviewer/default.nix

```diff
@@ -0,0 +1,51 @@
+{ stdenv
```

Comment:
It's very uncommon to put `stdenv` first. Many people recommend using `lib` first, and `stdenv` afterwards.